test/v2/client/test_client.go

```diff
@@ -365,28 +469,42 @@ func (c *TestClient) VerifyBlobCertification(
 	key corev2.BlobKey,
 	expectedQuorums []core.QuorumID,
 	signedBatch *v2.SignedBatch,
-	inclusionInfo *v2.BlobInclusionInfo) {
+	inclusionInfo *v2.BlobInclusionInfo) error {
```

Comment:
This method isn't being called anywhere, do you still need it?

Reply:
deleted